### DISCUSSION TITLE
The [redacted] topic seems to be missing

### DISCUSSION THREAD

**Discussion by AnIdiotsavant** [Monika-After-Story/MonikaModDev#7878] [QA - tech support + general] [answered]
I wanted to check out the floating islands again but the prompt seems to be missing

**Comment by Ulti70895** [comment]
Yes, it happens sometimes to me too, and seems to be at some specific time of the day that I cannot see the islands.

**Comment by KritRom** [comment, marked as answer]
The floating islands are only available when it's not sunset. Although, a sunset varient is being developed by the devs so hypee
Hope that answered your question
